bundles/org.eclipse.packagedrone.repo.adapter.rpm/src/org/eclipse/packagedrone/repo/adapter/rpm/internal/RpmSignerAspectFactory.java

```diff
@@ -1,5 +1,16 @@
 /*******************************************************************************
+ * Contract No:  FA8750-14-D-0037
```

Comment:
Could you please align that with the other headers:

e.g.:
~~~
Copyright (c) 2019 Trident Systems, Inc.
~~~

Reply:
Certainly. I've added that line to each file containing that header.

Reply:
I am sorry, but I was assuming to drop:
~~~
 * Contract No:  FA8750-14-D-0037
 * Contractor Name:   Trident Systems, Inc.
 * Contractor Address:  10201 Fairfax Blvd. Suite 300, Fairfax, VA
~~~
as well. So that the license headers match the existing license headers.

Reply:
Oh, I see. Sure I can do that as well.